app/src/main/java/com/siri/RearrangeScreens/FourthActivity.kt

```diff
@@ -0,0 +1,25 @@
+package com.siri.RearrangeScreens
+import android.os.Bundle
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+import android.app.Activity
+
+import android.R.attr.y
+
+import android.content.Intent
+
+
+
+
+class FourthActivity : AppCompatActivity() {
```

Comment:
Install a linter.  You are using whitespace in funny ways :)

You don't have to install one in this PR.  You could instead create a ticket for your next sprint to install one.

Reply:
Will install linter as part of a ticket in sprint as suggested